src/test/java/uk/gov/ons/ctp/integration/fieldsvccucumber/cucSteps/sso/TestConfig.java

```diff
@@ -2,9 +2,11 @@
 
 import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
 
 @Data
 @ConfigurationProperties("config")
+@Component
```

Comment:
unnecessary change. The ConfigurationProperties annotation is enough for Spring.

Reply:
Meh

Reply:
It does work without it but my jury is still out :-)